packages/eslint-plugin/lib/config/recommended.js

```diff
@@ -178,6 +178,15 @@ module.exports = {
         aliases: ['applications', 'platform', 'site', '@@vap-svc', '@@profile'],
       },
     ],
+    '@department-of-veterans-affairs/no-cross-app-imports': [
```

Comment:
Since cross app imports are specific to the `vets-website` repo, should we add this to the ESLint config in `vets-website` instead? `content-build` also uses the recommended config, and cross app imports don't apply there

Reply:
Yes good point, this config will only be needed in `vets-website`. I'll remove it from here and create a separate PR to add it there